taxcalc/functions.py

```diff
@@ -498,26 +503,30 @@ def StdDed(DSI, _earned, STD, e04470, e00100, e60000,
     c15200 = c04200
 
     # Compute the total standard deduction
+    _standard = c04100 + c04200
+
     if (MARS == 3 or MARS == 6) and (MIDR == 1):
         _standard = 0.
-    else:
-        _standard = c04100 + c04200
 
-    # ???
+    # c04500 = c00100 - max(c04470, max(c04100, _standard + e37717))
+    c04500 = c00100 - max(c21060 - c21040, _standard + e37717)
```

Comment:
It seems that the two evaluations are equivalent, note that `max(c04100, _standard + e37717) = _standard + e37717` is vacuously true.